FAIL: test_future_seconds on MacOS

Here's the error:

```
======================================================================
FAIL: test_future_seconds (tests.helper_test.TestPrettyDatetime)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/var/tmp/nix-build-python2.7-alot-0.5.1.drv-0/alot-0.5.1-src/tests/helper_test.py", line 248, in test_future_seconds
    self.assertEqual(actual, expected)
AssertionError: u'12:00pm' != u'12:00p'
- 12:00pm
?       -
+ 12:00p
```

full log is available here: http://hydra.nixos.org/build/53107329/nixlog/3

Thanks for reporting this. It should be easy to adjust this test.
Slightly surprising representation though :)
Cheers,
P

Quoting Matthew Justin Bauer (2017-05-24 20:51:00)
> Here's the error:
> 
> ======================================================================
> FAIL: test_future_seconds (tests.helper_test.TestPrettyDatetime)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/private/var/tmp/nix-build-python2.7-alot-0.5.1.drv-0/alot-0.5.1-src/tests/helper_test.py", line 248, in test_future_seconds
>     self.assertEqual(actual, expected)
> AssertionError: u'12:00pm' != u'12:00p'
> - 12:00pm
> ?       -
> + 12:00p
> 
> full log is available here: http://hydra.nixos.org/build/53107329/nixlog/3
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>

--- a/tests/helper_test.py
+++ b/tests/helper_test.py
@@ -243,7 +243,7 @@ class TestPrettyDatetime(unittest.TestCase):
     @staticmethod
     def _future_expected(test):
         if test.strftime('%p'):
-            expected = test.strftime('%I:%M%P').lower()
+            expected = test.strftime('%I:%M%p').lower()
         else:
             expected = test.strftime('%H:%M')
         expected = expected.decode('utf-8')
@@ -276,8 +276,8 @@ class TestPrettyDatetime(unittest.TestCase):
     def test_future_days(self):
         def make_expected():
             # Uses the hourfmt instead of the hourminfmt from pretty_datetime
-            if test.strftime('%P'):
-                expected = test.strftime('%I%P')
+            if test.strftime('%p'):
+                expected = test.strftime('%I%p')
             else:
                 expected = test.strftime('%Hh')
             expected = expected.decode('utf-8')

